lib/counterfeit/base_app.rb

```diff
@@ -0,0 +1,14 @@
+require 'sinatra/base'
+
+module Counterfeit
+  class BaseApp < Sinatra::Base
+
+    def self.site_logger
+      @@logger ||= Logger.new('/var/www/log/counterfeit')
```

Comment:
There's probably a smarter way to do this. I am open. This is assuming that we are always in a docker container. Since this shouldn't be used in production it should be fine.....

Reply:
i think this will be a problem, this is running on staging / dev / test and that file and / or dir structure doesn't exist.  If this is just for this repo and isn't used anywhere else (ie counterfeit testing purposes), will be fine.  If not you should be using railties

Reply:
Ok. I'll take a look. Thanks.